docs/rfcs/XeTile.md

```diff
@@ -140,6 +121,79 @@ A `tile_mma` variant without vector_c initialization.
 		tile<64x64xbf16>, index, index into tile <64x64xbf16>
 ```
 
+
+`atomic_rmw_tile` atomically reads, modifies, and writes back data to the memory specified by the tile.
+
+```mlir
+  %ret_value = XeTile.atomic_rmw “addf” %value, %tile:
+          vector<8x16xbf16>, tile<8x16xbf16> to vector<8x16xbf16>
+```
+XeTile.atomic_rmw reuses the arith dialect attribute, mlir::arith::AtomicRMWKindAttr.
+
+
+`tile_transpose` transpose a 2D vector. It has the same semantics as the vector.transpose, but restricts the vector dimension to 2D. 
+```mlir
+   %vector_a = XeTile.transpose_tile [1, 0] %vector_b: vector<64x32xfloat> into vector<32x64xfloat>  
+```
```

Comment:
The dimention restriction is not the motivation. 

The motivation is that XeTile goal is to experiment work-group level tile-based programming, so identify a subset of vector dialect which we must assign special wg attribute help us to clearly understand the scope of the dialect. 

If we just think subgroup level, we may or may not need them. But I think it is better to include them since their lowering needs special care.

Reply:
ok, got it.